cpu/stm32_common/periph/i2c_2.c

```diff
@@ -40,28 +42,30 @@
 #include "mutex.h"
 #include "pm_layered.h"
 
-#include "periph_conf.h"
-#include "periph/gpio.h"
 #include "periph/i2c.h"
+#include "periph/gpio.h"
+#include "periph_conf.h"
 
-#define ENABLE_DEBUG    (0)
+/* DEBUG statements in the init causes the code to lock up */
```

Comment:
This could be removed if #10615 is merged.

Reply:
Yup, will do if it is merged before this one, otherwise I will remove it with the other PR.

Reply:
@MrKevinWeiss any chance to remove this today? And then we can merge.

Reply:
Thought I forgot something, done!